app/Http/Controllers/PayloadHistoryController.php

```diff
@@ -73,9 +73,15 @@ public function destroy(PayloadHistory $history)
             $this->payloadHistoryRepository->delete($history->id);
 
             return redirect(route('history.index'))
-                    ->with('messageSuccess', 'This payload history successfully deleted');
+                ->with('messageSuccess', [
+                    'status' => 'Delete success',
+                    'message' => 'This payload history successfully deleted'
+                ]);
         } catch (Exception $exception) {
-            return redirect()->back()->with('messageFail', 'Delete failed. Something went wrong');
+            return redirect()->back()->with('messageFail', [
+                'status' => 'Delete failed',
+                'message' => 'Delete failed. Something went wrong'
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/PayloadHistoryController.php</br>**Line**: 83</br>Each line in an array declaration must end in a comma